### PR TITLE
android: style exit node picker per Material UI and add disable button

### PR DESCRIPTION
**Pull request by oxtoacart**
Updates #ENG-2911

### CODE REVIEW

**Comment by oxtoacart** [comment]
I need to do a bit more work on this ...

File: android/src/main/java/com/tailscale/ipn/ui/util/LoadingIndicator.kt [hunk 1]
```diff
@@ -35,12 +38,12 @@ object LoadingIndicator {
...
       val isLoading = loading.collectAsState().value
       if (isLoading) {
-        Box(Modifier.matchParentSize().background(Color.Gray.copy(alpha = 0.5f)))
+        Box(Modifier.clickable {}.matchParentSize().background(Color.Gray.copy(alpha = 0.5f)))
 
         Column(
             modifier = Modifier.fillMaxWidth(),
             horizontalAlignment = Alignment.CenterHorizontally) {
-              CircularProgressIndicator()
+              TailscaleLogoView(true, Modifier.size(72.dp))
```
Comment (barnstar) [inline android/src/main/java/com/tailscale/ipn/ui/util/LoadingIndicator.kt:46]: Ha... Nice.

File: android/src/main/java/com/tailscale/ipn/ui/util/ClickableOrGrayedOut.kt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+fun Modifier.clickableOrGrayedOut(
+    enabled: Boolean = true,
+    onClickLabel: String? = null,
+    role: Role? = null,
+    onClick: () -> Unit
+) =
+    if (enabled) {
+      clickable(onClickLabel = onClickLabel, role = role, onClick = onClick)
+    } else {
+      alpha(0.75f)
```
Comment (barnstar) [inline android/src/main/java/com/tailscale/ipn/ui/util/ClickableOrGrayedOut.kt:26]: This should probably be an explicit colour, but it's fine for now.  We'll see what UX says.

File: android/src/main/java/com/tailscale/ipn/ui/view/ExitNodePicker.kt [hunk 3]
```diff
@@ -116,36 +78,43 @@ fun ExitNodePicker(
...
-              Text(stringResource(R.string.offline))
+              Icon(Icons.Outlined.Check, contentDescription = stringResource(R.string.selected))
             }
           }
         })
   }
 }
 
+@Composable
+fun MullvadItem(nav: ExitNodePickerNav, count: Int, selected: Boolean) {
```
Comment (barnstar) [inline android/src/main/java/com/tailscale/ipn/ui/view/ExitNodePicker.kt:104]: Nit: We should probably use a more generic term than "Mullvad" - considering that these are just exit nodes and, in theory, could be provided by any number of sources.

Comment (agottardo) [reply android/src/main/java/com/tailscale/ipn/ui/view/ExitNodePicker.kt:104]: We say `Location-based exit nodes` on other platforms. Mullvad happens to be one of the options for it.

**Review by agottardo — APPROVED** [review]

**Pull request merged**